### ISSUE TITLE
Line/Scatter chart, dots are misplaced

### ISSUE THREAD

**Issue by JosefWN** [chartist-js/chartist#729]
Starting with version 0.9.6 and up, I'm having issues with dots getting misplaced in Line (scatter) charts. I have verified that my input data is correct.

My code looks similar to this (I always have 101 points on the x-axis, most undefined), the only difference being I pass a dummy labels object since it's required in 0.9.5:

``` javascript
new Chartist.Line('.ct-chart', {
  series: [[undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, 69.22, undefined,  undefined, undefined,  undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, 52, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined]]       
}, {
  showLine: false,
  axisX: {
    showLabel: false,
    showGrid: false,
    offset: 0
  },
  axisY: {
    showLabel: false,
    showGrid: false,
    offset: 0
  },
  fullWidth: true,
  chartPadding: {
    top: 5,
    right: 5,
    bottom: 5,
    left: 5
  },
  low: 0,
  high: 100
});
```

Just pasing this into the live examples on the Chartist site seems to work, but in my code it doesn't. Downgrading to 0.9.5 again solves my issues. It could be a ReactJS issue, or it could be an issue in Chartist.

**Comment by gionkunz** [comment]
Works for me as expected with 0.9.8

**Comment by JosefWN** [comment]
Stil doesn't work for me. Not with 0.9.8 and not with 0.10.1. All positions get shifted. This is actually blocking me from upgrading chartist since 0.9.5.

0.9.5:
<img width="247" alt="screen shot 2016-10-25 at 22 07 59" src="https://cloud.githubusercontent.com/assets/12534742/19702517/4dddf4e8-9b00-11e6-9ff9-1287dafef088.png">

0.10.1:
<img width="244" alt="screen shot 2016-10-25 at 22 08 35" src="https://cloud.githubusercontent.com/assets/12534742/19702516/4dda1648-9b00-11e6-8f32-a53dbfd97008.png">

This is only after upgrading. I'm not changing anything in my own code.

**Comment by gionkunz** [comment]
Can you maybe show the code?

**Comment by gionkunz** [comment]
Because the code youve posted is working fine http://jsbin.com/yevolilado/edit?output. Maybe something else is causing the issue?

**Comment by JosefWN** [comment]
Tried disabling all your CSS (and most of mine) to have a nearly naked version, but it looks the same. I tried offsetting the svg group and found that chartist seems to think that my container is bigger than it is, making it look like the points are misplaced. In fact, it's the scale of the background that is not the same as the scale of the plot area.

<img width="220" alt="screen shot 2016-11-08 at 23 55 57" src="https://cloud.githubusercontent.com/assets/12534742/20120982/6cb8e502-a60f-11e6-84b8-988440fac1c4.png">

EDIT: The issue seems to be my CSS transform applied to the parent of ct-chart:

> transform: translateX(-50%) translateY(-50%) rotate(-22deg);

Odd, it used to work... I'm open to suggestions on how to fix this. Being able to transform the parent is a pretty neat use-case for chartist.

It does work if I apply my transform after your transform has been applied (by commenting it out when the page is rendered, then enabling it in Chrome developer tools), suggesting that the two transforms occur in the wrong order?
